docs/release-notes.md

```diff
@@ -24,14 +50,6 @@ nav_order: 9
 - Stabilize r4e spec 1.1.0, targeting Ignition spec 3.4.0
 - Add r4e spec 1.2.0-experimental, targeting Ignition spec
```

Comment:
Usually we've spelled these out: Fedora CoreOS, Flatcar, RHEL for Edge.

Reply:
The latest push still has `fcos` and `flatcar`.

Reply:
IDK what I was thinking.